package.json

```diff
@@ -11,6 +11,9 @@
     "./globals": {
       "import": "./globals.mjs",
       "require": "./dist/globals.cjs"
+    },
```

Comment:
Allow import this file.

Don't need to support cjs usage, as it is going to work just for final zx.mjs